code/modules/hydroponics/grown.dm

```diff
@@ -46,13 +46,13 @@
 	make_dryable()
 
 	// Go through all traits in their genes and call on_new_plant from them.
-	for(var/datum/plant_gene/trait/trait in seed.genes)
+	for(var/datum/plant_gene/trait/trait in seed?.genes)
 		trait.on_new_plant(src, loc)
 
 	. = ..() //Only call it here because we want all the genes and shit to be applied before we add edibility. God this code is a mess.
 
-	seed.prepare_result(src)
-	transform *= TRANSFORM_USING_VARIABLE(seed.potency, 100) + 0.5 //Makes the resulting produce's sprite larger or smaller based on potency!
+	seed?.prepare_result(src)
+	transform *= TRANSFORM_USING_VARIABLE(seed?.potency, 100) + 0.5 //Makes the resulting produce's sprite larger or smaller based on potency!
```

Comment:
Do you know what happens to the transform if seed happens to be null in this line?

Also, which plant was failing the test for having a `null` seed? If there's a grown item with a null seed that might be a bigger problem

Reply:
It was just the bombanana with a null seed. I tested it, and while it just spawned a tiny banana, it created a runtime with eating it. I imagine other runtimes with other growns potentially. So I just decided not to check it at all. If it runtimes, people should fix it by adding a seed.